packages/mds-geography/tests/index.spec.ts

```diff
@@ -83,7 +76,7 @@ describe('Tests app', () => {
     })
 
     it('cannot GET a nonexistent geography', done => {
-      request
+      void request
```

Comment:
Can we convert these tests to `async`... If they are `async () =>` then you can change `void request` to `await request` and get rid of the `.end`.

Reply:
I like that idea